provider/aws/formation/resource/postgres.json.tmpl

```diff
@@ -31,13 +39,17 @@
       "Type": "Number",
       "Default": "20"
     },
+    "MasterUsername": {
```

Comment:
Is there any real advantage to parameterizing this since the result comes out in the URL?

Reply:
We have some apps that we will dump data into, which have roles already declared. This is for apps that we migrating and may have external clients that also connect to DBs.

Reply:
This one should be called `Username`